lib/videoroom/room.ex

```diff
@@ -77,9 +87,18 @@ defmodule Videoroom.Room do
       TURNManager.ensure_tls_turn_launched(integrated_turn_options, port: tls_turn_port)
     end
 
-    {:ok, pid} = Membrane.RTC.Engine.start(rtc_engine_options, [])
+    {:ok, pid} = Membrane.RTC.Engine.start_link(rtc_engine_options, [])
     Engine.register(pid, self())
-    Process.monitor(pid)
+
+    Task.Supervisor.start_child(
+      Videoroom.RoomMonitorSupervisor,
+      Videoroom.Room.Monitor,
+      :monitor,
+      [
+        self(),
+        room_id
+      ]
+    )
```

Comment:
That's not a big issue, but Tasks are not meant to be long-lasting processes, using `Task` and `Task.Supervisor` here seems to be against their semantics.
I'd recommend using a `DynamicSupervisor` instead of `Task.Supervisor` and converting `Room.Monitor` to a process either by using `GenServer` or implementing `child_spec/1` + start_link